main.go

```diff
@@ -9,6 +9,9 @@ import (
 
 var bot *tbot.BotAPI
 
+func intro(ID int64){
```

Comment:
Remove the functions that aren't usable right now. Go doesn't allow function definition that weren't used at all.

Reply:
Ok.